conference.js

```diff
@@ -499,6 +499,11 @@ export default {
                     mediaPermissionPromptVisibilityChanged(true, browserName))
         );
 
+        JitsiMeetJS.mediaDevices.addEventListener(
+            JitsiMediaDevicesEvents.SLOW_GET_USER_MEDIA,
```

Comment:
there's no such event `JitsiMediaDevicesEvents.SLOW_GET_USER_MEDIA` right now. Is there a PR for lib-jitsi-meet? https://github.com/jitsi/lib-jitsi-meet/blob/master/JitsiMediaDevicesEvents.js

Reply:
https://github.com/jitsi/lib-jitsi-meet/pull/1406